addons/website/static/src/components/dialog/add_page_dialog.js

```diff
@@ -359,15 +361,30 @@ export class AddPageTemplates extends Component {
         const activeTabEl = this.tabsRef.el.querySelector(".active");
         const activePaneEl = this.panesRef.el.querySelector(".active");
         activeTabEl?.classList?.remove("active");
+        activeTabEl?.setAttribute("tabIndex", -1);
         activePaneEl?.classList?.remove("active");
         activePaneEl?.setAttribute("inert", true); // Make sure trapFocus() works.
         const tabEl = this.tabsRef.el.querySelector(`[data-id=${id}]`);
         const paneEl = this.panesRef.el.querySelector(`[data-id=${id}]`);
         tabEl.classList.add("active");
+        tabEl.tabIndex = 0;
         paneEl.classList.add("active");
         paneEl.removeAttribute("inert");
         this.props.onTemplatePageChanged(tabEl.dataset.id === "basic" ? "" : tabEl.textContent);
     }
+
+    onTabKeydown(ev) {
+        const hotkey = getActiveHotkey(ev);
```

Comment:
Same question: should not it be keyup? Plus I would not call it "tab key down" while you don't know if it is a tab.